### PR TITLE
Update 'what is a web server'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

part of #13640

### CODE REVIEW

File: files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md [hunk 1]
```diff
@@ -1,107 +1,106 @@
...
+在这篇文章中我们会重温什么是 web 服务器，它们如何工作，以及为什么它们很重要。
+
+<table>
   <tbody>
     <tr>
       <th scope="row">前提：</th>
       <td>
         你应该已经知道
-        <a href="/zh-CN/docs/Learn/How_the_Internet_works">互联网是怎么工作的</a
+        <a href="/zh-CN/docs/Learn/Common_questions/Web_mechanics/How_the_Internet_works">互联网是怎么工作的</a
```
Comment (jasonren0403) [inline files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md:16]:
```suggestion
        <a href="/zh-CN/docs/Learn/Common_questions/Web_mechanics/How_does_the_Internet_work">互联网是怎么工作的</a
```

File: files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md [hunk 2]
```diff
@@ -1,107 +1,106 @@
...
+在这篇文章中我们会重温什么是 web 服务器，它们如何工作，以及为什么它们很重要。
+
+<table>
   <tbody>
     <tr>
       <th scope="row">前提：</th>
       <td>
         你应该已经知道
-        <a href="/zh-CN/docs/Learn/How_the_Internet_works">互联网是怎么工作的</a
+        <a href="/zh-CN/docs/Learn/Common_questions/Web_mechanics/How_does_the_Internet_work">互联网是怎么工作的</a
```
Comment (yin1999) [inline files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md:16]:
```suggestion
        你应该已经知道<a
          href="/zh-CN/docs/Learn/Common_questions/Web_mechanics/How_does_the_Internet_work">互联网是怎么工作的</a
```

File: files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md [hunk 3]
```diff
@@ -1,107 +1,106 @@
...
         >，并且<a
-          href="/zh-CN/docs/Learn/Common_questions/Pages_sites_servers_and_search_engines"
-          >了解网页，网站，网络服务器和搜索引擎的区别是什么</a
+          href="/zh-CN/docs/Learn/Common_questions/Web_mechanics/Pages_sites_servers_and_search_engines"
+          >了解网页、网站、web 服务器和搜索引擎的区别是什么</a
         >
```
Comment (yin1999) [inline files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md:20]:
```suggestion
        >。
```

File: files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md [hunk 4]
```diff
@@ -1,107 +1,106 @@
...
-      <td>了解网络服务器，明白它大概的工作原理</td>
+      <td>了解 web 服务器，明白它大概的工作原理</td>
     </tr>
   </tbody>
 </table>
 
 ## 概述
 
-“网络服务器（Web server）”可以代指硬件或软件，或者是它们协同工作的整体。
+*网络服务器*（Web server）一词可以代指硬件或软件，或者是它们协同工作的整体。
```
Comment (yin1999) [inline files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md:32]:
```suggestion
*web 服务器*一词可以代指硬件或软件，或者是它们协同工作的整体。
```

File: files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md [hunk 5]
```diff
@@ -1,107 +1,106 @@
...
 
-第二点，一个网络服务器提供了 {{Glossary("HTTP")}}（超文本传输协议）支持。正如它的名字暗示，HTTP 明确提出了如何在两台计算机间传输超文本（比如说链接的网络文档（linked web documents））。
+其次，web 服务器提供了 {{Glossary("HTTP")}}（**H**yper**t**ext **T**ransfer **P**rotocol，超文本传输协议）支持。正如它的名字暗示，HTTP 明确提出了如何在两台计算机间传输超文本（链接的 web 文档）。
 
-一个*{{Glossary("Protocol", "协议")}}*是一套为了在两台计算机间交流而制定的规则。HTTP 是一个文本化的（textual），无状态的（stateless）协议。
+{{Glossary("Protocol", "协议")}}是一套为了在两台计算机间交流而制定的规则。HTTP 是一个文本化的（textual），无状态的（stateless）协议。
 
 - 文本化
-  - : 所有的命令都是纯文本的（plain-text）和人类可读的（human-readable）。
+  - : 所有的命令都是纯文本的（plain-text），人类可读（human-readable）。
```
Comment (yin1999) [inline files/zh-cn/learn/common_questions/web_mechanics/what_is_a_web_server/index.md:75]:
```suggestion
  - : 所有的命令都是纯文本（plain-text）且人类可读（human-readable）的。
```

**Review by yin1999 — APPROVED** [review]
Thanks for the quick fixes, I've some suggestions about this PR. Overwise, feel free to merge this :)

**Pull request merged**